### PR TITLE
test: let's merge more tests together

### PR DESCRIPTION
**Pull request by mrc0mmand**
Let's do another round of merging tests together to further save some precious time in CIs. There's still a couple of candidates for further merging, but I'd like to go with "just" this batch to see if anything breaks along the way.

### CODE REVIEW

File: test/units/testsuite-23.clean-unit.sh [hunk 1]
```diff
@@ -114,6 +125,7 @@ test -L /var/log/test-service
 
 systemctl stop test-service
 
+
```
Comment (yuwata) [inline test/units/testsuite-23.clean-unit.sh]: Why this empty line inserted?

File: test/units/testsuite-23.clean-unit.sh [hunk 2]
```diff
@@ -261,6 +273,7 @@ EOF
 
 systemctl daemon-reload
 
+
```
Comment (yuwata) [inline test/units/testsuite-23.clean-unit.sh]: Also here.

**Review by yuwata — APPROVED** [review]

**Comment by yuwata** [comment]
Suggestions (not necessary in this PR):
- `runas()` can be moved to util.sh.
- TEST-60-MOUNT-RATELIMIT -> TEST-60-MOUNT

File: test/units/testsuite-23.clean-unit.sh (hunk 1 above)
Comment (mrc0mmand) [reply test/units/testsuite-23.clean-unit.sh]: Whoops :facepalm:

**Comment by mrc0mmand** [comment]
> Suggestions (not necessary in this PR):
> 
>     * `runas()` can be moved to util.sh.
> 
>     * TEST-60-MOUNT-RATELIMIT -> TEST-60-MOUNT

Nice, thanks! I moved the `runas()` function straight away and I'll do the TEST-60 rename (and possible merge) in the next test merge-fest.

File: test/units/testsuite-23.clean-unit.sh (hunk 1 above)
Comment (bluca) [reply test/units/testsuite-23.clean-unit.sh]: still here

Comment (mrc0mmand) [reply test/units/testsuite-23.clean-unit.sh]: *sigh*, I think GH mocks me today. I'm pretty sure I finally fixed it (attempt \#3), but now I can't push it:

```
$ git push -f origin test-merge 
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.
```

**Review by bluca — APPROVED** [review]

**Pull request merged**